### PR TITLE
Add the rest of the badges

### PR DESCRIPTION
**Pull request by radinat**

### CODE REVIEW

File: packages/config/src/projects/layer2s/aevo.ts [hunk 1]
```diff
@@ -2,12 +2,14 @@ import { EthereumAddress, UnixTime } from '@l2beat/shared-pure'
...
 import { subtractOne } from '../../common/assessCount'
 import { ProjectDiscovery } from '../../discovery/ProjectDiscovery'
+import { Badge } from '../badges'
 import { CELESTIA_DA_PROVIDER, opStackL2 } from './templates/opStack'
 import { Layer2 } from './types'
 
 const discovery = new ProjectDiscovery('aevo')
 
 export const aevo: Layer2 = opStackL2({
+  badges: [Badge.VM.EVM, Badge.DA.Celestia, Badge.Stack.OPStack],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer2s/aevo.ts:12]: pls add conduit as a raas to conduit chains, check either https://rollup.wtf/ or https://app.conduit.xyz/browse-networks

File: packages/config/src/projects/layer2s/ancient.ts [hunk 2]
```diff
@@ -13,6 +14,7 @@ const upgradeability = {
 
 export const ancient: Layer2 = opStackL2({
   daProvider: CELESTIA_DA_PROVIDER,
+  badges: [Badge.VM.EVM, Badge.Stack.OPStack, Badge.DA.Celestia],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer2s/ancient.ts:17]: could be added in the template

File: packages/config/src/projects/layer2s/cyber.ts [hunk 3]
```diff
@@ -23,6 +24,7 @@ const daResolveWindow = formatSeconds(
 export const cyber: Layer2 = opStackL2({
   associatedTokens: ['CYBER'],
   discovery,
+  badges: [Badge.VM.EVM, Badge.Stack.OPStack],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer2s/cyber.ts:27]: custom DA would be a nice badge

File: packages/config/src/projects/layer2s/fraxtal.ts [hunk 4]
```diff
@@ -16,6 +17,7 @@ const upgradeability = {
 }
 
 export const fraxtal: Layer2 = opStackL2({
+  badges: [Badge.VM.EVM, Badge.Stack.OPStack],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer2s/fraxtal.ts:20]: custom DA?

File: packages/config/src/projects/layer2s/fuelv1.ts [hunk 5]
```diff
@@ -11,12 +11,14 @@ import {
...
   makeBridgeCompatible,
 } from '../../common'
+import { Badge } from '../badges'
 import { getStage } from './common/stages/getStage'
 import { Layer2 } from './types'
 
 export const fuelv1: Layer2 = {
   type: 'layer2',
   id: ProjectId('fuelv1'),
+  badges: [Badge.VM.FuelVM, Badge.DA.EthereumCalldata],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer2s/fuelv1.ts:21]: fuelvm is usually referred to fuel v2, i wouldn't use it here

File: packages/config/src/projects/layer2s/honeypot.ts [hunk 6]
```diff
@@ -19,6 +20,7 @@ const discovery = new ProjectDiscovery('honeypot')
 export const honeypot: Layer2 = {
   type: 'layer2',
   id: ProjectId('honeypot'),
+  badges: [Badge.VM.EVM, Badge.DA.EthereumCalldata],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer2s/honeypot.ts:23]: cartesi stack

File: packages/config/src/projects/layer2s/hychain.ts [hunk 7]
```diff
@@ -1,10 +1,12 @@
 import { ProjectDiscovery } from '../../discovery/ProjectDiscovery'
+import { Badge } from '../badges'
 import { orbitStackL2 } from './templates/orbitStack'
 import { Layer2 } from './types'
 
 const discovery = new ProjectDiscovery('hychain', 'ethereum')
 
 export const hychain: Layer2 = orbitStackL2({
+  badges: [Badge.VM.EVM, Badge.Stack.Orbit],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer2s/hychain.ts:9]: DAC for DA

File: packages/config/src/projects/layer2s/mantle.ts [hunk 8]
```diff
@@ -23,6 +24,7 @@ const threshold =
   ) / 1000 // Quorum threshold is in basis points, but stake is equal for all members (100k MNT)
 
 export const mantle: Layer2 = opStackL2({
+  badges: [Badge.VM.EVM, Badge.Stack.OPStack],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer2s/mantle.ts:27]: custom DA

File: packages/config/src/projects/layer2s/myria.ts [hunk 9]
```diff
@@ -56,6 +57,7 @@ const committee = getCommittee(discovery)
 export const myria: Layer2 = {
   type: 'layer2',
   id: ProjectId('myria'),
+  badges: [Badge.VM.CairoVM, Badge.RaaS.StarkEx, Badge.Infra.SHARP],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer2s/myria.ts:60]: not CairoVM, and StarkEx is a stack not a RaaS

File: packages/config/src/projects/layer2s/nova.ts [hunk 10]
```diff
@@ -43,6 +44,7 @@ const maxTimeVariation = discovery.getContractValue<number[]>(
 const selfSequencingDelay = maxTimeVariation[2]
 
 export const nova: Layer2 = orbitStackL2({
+  badges: [Badge.VM.EVM, Badge.Stack.Orbit],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer2s/nova.ts:47]: DAC

File: packages/config/src/projects/layer2s/real.ts [hunk 11]
```diff
@@ -1,13 +1,15 @@
...
 import { ProjectDiscovery } from '../../discovery/ProjectDiscovery'
+import { Badge } from '../badges'
 import { orbitStackL2 } from './templates/orbitStack'
 import { Layer2 } from './types'
 
 const discovery = new ProjectDiscovery('real')
 
 export const real: Layer2 = orbitStackL2({
   discovery,
+  badges: [Badge.VM.EVM, Badge.Stack.Orbit],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer2s/real.ts:12]: DAC

File: packages/config/src/projects/layer2s/reddioex.ts [hunk 12]
```diff
@@ -50,6 +51,7 @@ const committee = getCommittee(discovery)
 export const reddioex: Layer2 = {
   type: 'layer2',
   id: ProjectId('reddioex'),
+  badges: [Badge.VM.CairoVM, Badge.RaaS.StarkEx, Badge.Infra.SHARP],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer2s/reddioex.ts:54]: not CairoVM and StarkEx is a stack

File: packages/config/src/projects/layer2s/redstone.ts [hunk 13]
```diff
@@ -22,6 +23,7 @@ const daResolveWindow = formatSeconds(
 
 export const redstone: Layer2 = opStackL2({
   discovery,
+  badges: [Badge.VM.EVM, Badge.Stack.Orbit],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer2s/redstone.ts:26]: it's op stack not orbit, and custom DA

File: packages/config/src/projects/layer2s/reya.ts [hunk 14]
```diff
@@ -1,13 +1,15 @@
...
 import { ProjectDiscovery } from '../../discovery/ProjectDiscovery'
+import { Badge } from '../badges'
 import { orbitStackL2 } from './templates/orbitStack'
 import { Layer2 } from './types'
 
 const discovery = new ProjectDiscovery('reya')
 
 export const reya: Layer2 = orbitStackL2({
   discovery,
+  badges: [Badge.VM.EVM, Badge.Stack.Orbit],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer2s/reya.ts:12]: DAC

File: packages/config/src/projects/layer2s/rhinofi.ts [hunk 15]
```diff
@@ -53,6 +54,7 @@ const committee = getCommittee(discovery)
 
 export const rhinofi: Layer2 = {
   type: 'layer2',
+  badges: [Badge.VM.CairoVM, Badge.RaaS.StarkEx, Badge.Infra.SHARP],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer2s/rhinofi.ts:57]: not CairoVM and StarkEx is a stack

File: packages/config/src/projects/layer2s/sorare.ts [hunk 16]
```diff
@@ -53,6 +54,7 @@ const committee = getCommittee(discovery)
 
 export const sorare: Layer2 = {
   type: 'layer2',
+  badges: [Badge.VM.CairoVM, Badge.RaaS.StarkEx, Badge.Infra.SHARP],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer2s/sorare.ts:57]: not CairoVM and StarkEx is a stack

File: packages/config/src/projects/layer2s/tanx.ts [hunk 17]
```diff
@@ -55,6 +56,7 @@ const committee = getCommittee(discovery)
 export const tanx: Layer2 = {
   type: 'layer2',
   id: ProjectId('brine'),
+  badges: [Badge.VM.CairoVM, Badge.RaaS.StarkEx, Badge.Infra.SHARP],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer2s/tanx.ts:59]: not CairoVM and StarkEx is a stack

File: packages/config/src/projects/layer2s/zkfair.ts [hunk 18]
```diff
@@ -87,6 +88,7 @@ const requiredSignatures = discovery.getContractValue<number>(
 
 export const zkfair: Layer2 = {
   type: 'layer2',
+  badges: [Badge.VM.EVM, Badge.Stack.PolygonCDK],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer2s/zkfair.ts:91]: DAC

File: packages/config/src/projects/layer2s/zkspace.ts [hunk 19]
```diff
@@ -32,6 +33,7 @@ const upgradeability = {
 export const zkspace: Layer2 = {
   type: 'layer2',
   id: ProjectId('zkspace'),
+  badges: [Badge.VM.EVM],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer2s/zkspace.ts:36]: not EVM, zksync lite fork

File: packages/config/src/projects/layer2s/zksyncera.ts [hunk 20]
```diff
@@ -87,6 +88,13 @@ const validators = () => {
 export const zksyncera: Layer2 = {
   type: 'layer2',
   id: ProjectId('zksync2'),
+  badges: [
+    Badge.VM.EVM,
+    Badge.DA.EthereumBlobs,
+    Badge.Stack.ZKStack,
+    Badge.Other.Governance,
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer2s/zksyncera.ts:95]: doesnt really have governance yet

File: packages/config/src/projects/layer2s/zksyncera.ts [hunk 21]
```diff
@@ -87,6 +88,13 @@ const validators = () => {
...
+    Badge.Other.AccountAbstraction,
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer2s/zksyncera.ts:96]: i wouldnt put account abstraction badge before we first define it and then properly assess everyone

File: packages/config/src/projects/layer3s/degen.ts [hunk 22]
```diff
@@ -10,6 +11,7 @@ const discovery = new ProjectDiscovery('degen', 'base')
 export const degen: Layer3 = orbitStackL3({
   hostChain: ProjectId('base'),
   discovery,
+  badges: [Badge.VM.EVM, Badge.Stack.Orbit, Badge.L3ParentChain.Base],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer3s/degen.ts:14]: DAC

File: packages/config/src/projects/layer3s/deri.ts [hunk 23]
```diff
@@ -1,12 +1,14 @@
...
 
 import { ProjectDiscovery } from '../../discovery/ProjectDiscovery'
+import { Badge } from '../badges'
 import { orbitStackL3 } from '../layer2s/templates/orbitStack'
 
 const discovery = new ProjectDiscovery('deri', 'arbitrum')
 
 export const deri = orbitStackL3({
   discovery,
+  badges: [Badge.VM.EVM, Badge.Stack.Orbit, Badge.L3ParentChain.Arbitrum],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer3s/deri.ts:11]: blobs

File: packages/config/src/projects/layer3s/molten.ts [hunk 24]
```diff
@@ -1,12 +1,14 @@
...
 import { ProjectDiscovery } from '../../discovery/ProjectDiscovery'
+import { Badge } from '../badges'
 import { orbitStackL3 } from '../layer2s/templates/orbitStack'
 import { Layer3 } from './types'
 
 const discovery = new ProjectDiscovery('molten', 'arbitrum')
 
 export const molten: Layer3 = orbitStackL3({
   discovery,
+  badges: [Badge.VM.EVM, Badge.Stack.Orbit, Badge.L3ParentChain.Arbitrum],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer3s/molten.ts:11]: DAC

File: packages/config/src/projects/layer3s/popapex.ts [hunk 25]
```diff
@@ -2,12 +2,14 @@ import { assert, ProjectId } from '@l2beat/shared-pure'
...
 import { subtractOne } from '../../common/assessCount'
 import { ProjectDiscovery } from '../../discovery/ProjectDiscovery'
+import { Badge } from '../badges'
 import { orbitStackL3 } from '../layer2s/templates/orbitStack'
 import { Layer3 } from './types'
 
 const discovery = new ProjectDiscovery('popapex', 'arbitrum')
 
 export const popapex: Layer3 = orbitStackL3({
+  badges: [Badge.VM.EVM, Badge.Stack.Orbit, Badge.L3ParentChain.Arbitrum],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer3s/popapex.ts:12]: DAC

File: packages/config/src/projects/layer3s/rari.ts [hunk 26]
```diff
@@ -1,12 +1,14 @@
...
 
 import { ProjectDiscovery } from '../../discovery/ProjectDiscovery'
+import { Badge } from '../badges'
 import { orbitStackL3 } from '../layer2s/templates/orbitStack'
 import { Layer3 } from './types'
 
 const discovery = new ProjectDiscovery('rari', 'arbitrum')
 
 export const rari: Layer3 = orbitStackL3({
+  badges: [Badge.VM.EVM, Badge.Stack.Orbit, Badge.L3ParentChain.Arbitrum],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer3s/rari.ts:11]: DAC

File: packages/config/src/projects/layer3s/sanko.ts [hunk 27]
```diff
@@ -1,12 +1,14 @@
...
 import { ProjectDiscovery } from '../../discovery/ProjectDiscovery'
+import { Badge } from '../badges'
 import { orbitStackL3 } from '../layer2s/templates/orbitStack'
 import { Layer3 } from './types'
 
 const discovery = new ProjectDiscovery('sanko', 'arbitrum')
 
 export const sanko: Layer3 = orbitStackL3({
   discovery,
+  badges: [Badge.VM.EVM, Badge.Stack.Orbit, Badge.L3ParentChain.Arbitrum],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer3s/sanko.ts:11]: DAC

File: packages/config/src/projects/layer3s/syndicateframe.ts [hunk 28]
```diff
@@ -1,13 +1,15 @@
...
 import { ProjectDiscovery } from '../../discovery/ProjectDiscovery'
+import { Badge } from '../badges'
 import { opStackL3 } from '../layer2s/templates/opStack'
 import { Layer3 } from './types'
 
 const discovery = new ProjectDiscovery('syndicateframe', 'base')
 
 export const syndicateframe: Layer3 = opStackL3({
   discovery,
+  badges: [Badge.VM.EVM, Badge.Stack.OPStack, Badge.L3ParentChain.Base],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer3s/syndicateframe.ts:12]: blobs

File: packages/config/src/projects/layer3s/xai.ts [hunk 29]
```diff
@@ -32,6 +33,7 @@ assert(
 export const xai: Layer3 = orbitStackL3({
   discovery,
   hostChain: ProjectId('arbitrum'),
+  badges: [Badge.VM.EVM, Badge.Stack.Orbit, Badge.L3ParentChain.Arbitrum],
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer3s/xai.ts:36]: DAC

**Review by lucadonnoh — CHANGES_REQUESTED** [review]

File: packages/config/src/projects/layer2s/ancient.ts (hunk 2 above)
Comment (radinat) [reply packages/config/src/projects/layer2s/ancient.ts]: Need dev help for this one, will be done separately

**Review by lucadonnoh — APPROVED** [review]

**Review by lucadonnoh — APPROVED** [review]

**Pull request merged**